servers/server.js

```diff
@@ -0,0 +1,27 @@
+const http = require('http');
+const url = require('url');
+const airport = require('../src/airportDetailsRetriever');
+
+const host = 'localhost';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  const queries = url.parse(req.url, true).query;
+  
+  if ('airport-code' in queries) {
```

Comment:
No endpoint defined?
Right now any random endpoint would be served if I pass it query parameter `?airport-code=XYZ`
Try http://localhost:3000/abcdefghi?airport-code=MAS or http://localhost:3000/abcdefghi?airport-code=XYZ